### DISCUSSION TITLE
Does this work?

### DISCUSSION THREAD

**Discussion by xsploit** [flowese/UdioWrapper#15] [General]
Does this still work?

**Comment by danlimaverde** [comment]
I also would like to know

**Comment by erokone1** [comment]
sb-api-auth does not exist in cookies atm. tried with sb-ssr-production-auth-token.0 and sb-ssr-production-auth-token.1 (api's from cookies) and it gave me an error: Error making POST request to https://www.udio.com/api/generate-proxy: 401 Client Error: Unauthorized for url: https://www.udio.com/api/generate-proxy
